src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -200,7 +229,30 @@ int halide_hexagon_remote_power_hvx_on() {
 int halide_hexagon_remote_power_hvx_off() {
     power_ref_count--;
     if (power_ref_count == 0) {
-        HAP_power_request(0, 0, -1);
+        HAP_power_request_t request;
+
+        request.type = HAP_power_set_HVX;
+        request.hvx.power_up = FALSE;
+        int retval = HAP_power_set(NULL, &request);
```

Comment:
Seems like the easiest way to test this is to insert a call to halide_hexagon_remote_power_hvx_off() right at the end of the code that powers it on, then run the benchmark with that change and see if it fails.  My test systems are in process of getting an upgrade so it might take me awhile to get to try this.